sample_apps/run_example.sh

```diff
@@ -38,7 +38,7 @@ MeMsg="Build and run example program"
 
 # Script global symbols to app-data dirs
 Client_app_data="app1_data"
-Srvr_app_data="app2_data"
+Server_app_data="app2_data"
```

Comment:
Variable name-diff ... Ignore this in the rest of this file.

I've annotated other diffs in this file that merit an 2nd look ...